pkgs/applications/editors/emacs-modes/org-packages.nix

```diff
@@ -0,0 +1,42 @@
+/*
+
+# Updating
+
+To update the list of packages from ELPA,
+
+1. Clone https://github.com/ttuegel/emacs2nix
+2. Run `./org-packages.sh` from emacs2nix
+3. Copy the new elpa-packages.json file into Nixpkgs
+4. `git commit -m "elpa-packages $(date -Idate)"`
+
+*/
+
+{ fetchurl, lib, stdenv, texinfo }:
+
+self:
+
+  let
+
+    imported = import ./org-generated.nix {
+      inherit (self) callPackage;
+    };
+
+    super = imported;
+
+    elpaBuild = import ../../../build-support/emacs/elpa.nix {
```

Comment:
@mdorman You can also remove `elpaBuild` here, since it is not referenced. The only references are to `self.elpaBuild`, which will be inherited from `elpaPackages`.